module/Activity/src/Activity/Service/Signup.php

```diff
@@ -87,10 +87,14 @@ public function getSignedUpData(ActivityModel $activity)
         }
 
         $fieldValueMapper = $this->getServiceManager()->get('activity_mapper_activity_field_value');
+        $memberMapper = $this->getServiceManager()->get('decision_mapper_member');
         $result = [];
         foreach($activity->getSignUps() as $signup){
             $entry = [];
-            $entry['member'] = $signup->getUser()->getMember()->getFullName();
+            //$signup->getUser()->getMember() yields an empty result if the signup is new.
+            //(i.e. a signup has taken place in the same request)
+            $member = $memberMapper->findByLidnr($signup->getUser()->getLidnr());
```

Comment:
I cannot reproduce the issues I got myself either. Which version should be kept? ('either one' is a valid answer)